packages/api-client/src/api/products/products.ts

```diff
@@ -1,9 +1,9 @@
 import gql from 'graphql-tag';
-import { MediaImagesFragment, AttributeFragment, InventoryFragment, ProductPriceFragment } from '../../fragments';
+import { MediaImagesFragment, AttributeValueFragment, InventoryFragment, ProductPriceFragment } from '../../fragments';
 
 export default gql`
   ${MediaImagesFragment}
-  ${AttributeFragment}
+  ${AttributeValueFragment}
   ${InventoryFragment}
   ${ProductPriceFragment}
   query products(
```

Comment:
Please remove all the deprecated fields from the query and remap them where needed in the composables.

for example `product.classId`, `Crossupsell.product` , `Crossupsell.product.classId`, etc. etc.

Note: all deprecated fields will be highlighted in the Playground when preparing the queries.